services/web/server/src/simcore_service_webserver/payments/_methods_api.py

```diff
@@ -79,7 +79,7 @@ async def _fake_init_creation_of_wallet_payment_method(
     await asyncio.sleep(1)
     payment_method_id = PaymentMethodID(f"{_FAKE_PAYMENT_METHOD_ID_PREFIX}_{uuid4()}")
     form_link = (
-        URL(settings.PAYMENTS_FAKE_GATEWAY_URL)
+        URL(f"{settings.PAYMENTS_FAKE_GATEWAY_URL}")
```

Comment:
THOUGHT: maybe we should have some helpers with converters from/to all URL classes we have. From the top of my head we are using at least these
- yarl.URL
- pydantic.HTTPUrl
- httpx.URL
- starlette.URL (via fastapi)